Documentation/gettingstarted/cli-download.rst

```diff
@@ -1,13 +1,15 @@
-Install the latest version of the Cilium CLI on your local machine. The Cilium
-CLI can be used to install Cilium, inspect the state of a Cilium installation,
-and enable/disable a variety of functionality.
+Install the latest version of the Cilium CLI. The Cilium CLI can be used to
+install Cilium, inspect the state of a Cilium installation, and enable/disable
+a variety of functionality.
```

Comment:
functionality=>functionalities?

Reply:
How about this instead? I hadn't touched the initial text but if we're changing it, I believe this sounds more accurate?
```suggestion
various features (e.g. clustermesh, Hubble).
```

Reply:
Fixed in latest push.